cdk/src/open-data-platform/data-plane/data-import/write-water-systems-data.handler.ts

```diff
@@ -85,25 +94,23 @@ function getTableRowFromRow(row: any): SqlParametersList | null {
     return null;
   }
 
-  const reportedLeadConnections = getValueOrDefault(properties.lead_connections_count);
-  let leadConnectionsCount;
+  let leadConnectionsCount = getValueOrDefault(properties.lead_connections_count);
 
   // If there is a reported lead count, use that
-  if (reportedLeadConnections > 0) {
-    leadConnectionsCount = reportedLeadConnections;
-    // Otherwise, get estimate from prediction interval.
-  } else {
+  const leadCountIsNotReported = leadConnectionsCount == null || leadConnectionsCount < 0;
+  const lowEstimate = getValueOrDefault(properties.low);
+  const highEstimate = getValueOrDefault(properties.high);
+  if (leadCountIsNotReported && lowEstimate != null && highEstimate != null) {
     // TODO: Switch to storing low, high in db.
-    leadConnectionsCount =
-      (getValueOrDefault(properties.low) + getValueOrDefault(properties.high)) / 2;
+    leadConnectionsCount = (lowEstimate + highEstimate) / 2;
   }
   return (
     new WaterSystemsTableRowBuilder()
       .pwsId(properties.pwsid)
       .pwsName(properties.pws_name ?? '')
       .leadConnectionsCount(leadConnectionsCount)
       .serviceConnectionsCount(getValueOrDefault(properties.service_connections_count))
-      .populationServed(getValueOrDefault(properties.population_served))
+      .populationServed(getValueOrDefault(properties.population_served) ?? 0)
```

Comment:
Why do we want population to default to 0 if there's no data?

Reply:
Good catch. This value is already nullable so didn't need this ?? 0